content/release-10-17-0/announcements-10-17-0.md

```diff
@@ -49,3 +49,7 @@ See also the announcement in the Streaming Analytics release notes for
 Existing blocks that use the version 1 API must be migrated to use the version 2 API.
 See [Migrating input and output blocks to the version 2 API](https://github.com/SoftwareAG/apama-analytics-builder-block-sdk/blob/rel/10.17.0.x/doc/150-MigrateInputOutputBlocks.md)
 in the Analytics Builder Block SDK documentation on GitHub for more details.
+
+##### Changes to microservice health endpoint
+
+For reasons of security and performance, the REST endpoint `/service/cep/health` no longer returns a comprehensive list of status values. All of the same information is still available from REST endpoints under `/service/cep/diagnostics/...`.
```

Comment:
Wondering if we really need the 3 dots after "diagnostics/". I'd leave them out.

Reply:
My thinking was that I don't want to give the impression that there is a specific endpoint called `/service/cep/diagnostics`, and I'm actually trying to say "every endpoint that starts with that".

If you still think I should remove the '...' though, I'm fine.

Reply:
That's why I did not remove the / from my suggestion ;-)
However, if @BeateRixen doesn't mind, we can keep the 3 dots.

Reply:
Fine with me :-).